QuickBytes/src/main/java/com/quickbytes/controller/VendorController.java

```diff
@@ -0,0 +1,13 @@
+package com.quickbytes.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.quickbytes.repository.VendorRepository;
+
+@RestController
+public class VendorController {
```

Comment:
Jay's Task, waiting for Jays PR to be accepted

Reply:
I don't see a PR up from Jay yet?